modules/representation_management/app/controllers/representation_management/v0/pdf_generator_base_controller.rb

```diff
@@ -31,6 +32,17 @@ def claimant_params_permitted
         ]
       end
 
+      def representative_params_permitted
```

Comment:
It might make sense to also have the representative id

Reply:
Added!